### PR TITLE
Adding instructions to use the sbs backup anywhere api

### PR DESCRIPTION
**Pull request by kyle-miller-centurylink**

### CODE REVIEW

File: Backup/backup-anywhere.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {{{
```
Comment (chrislittle) [inline Backup/backup-anywhere.md]: Shouldn't API documentation be with all other API docs in https://www.ctl.io/api-docs/v2/? not distributed out by product in kbs

Comment (kyle-miller-centurylink) [reply Backup/backup-anywhere.md]: I didn't realize that was there. That will need updated as well. I will add section(s) to that as well.

Comment (kyle-miller-centurylink) [reply Backup/backup-anywhere.md]: I've created a pull request to add the API documentation for retrieving anywhere servers to https://github.com/CenturyLinkCloud/APIDocs as well

File: Backup/backup-anywhere.md [hunk 2]
```diff
@@ -41,4 +41,62 @@ There are 2 methods available to install the Simple Backup agent on your non-Cen
...
+    {
+    	"id":"UUID",
+    	"accountAlias":"ABCD",
+    	"osType":"selected ostype",
+    	"serverAlias":"your-submitted-server-alias"
+    }
+]
+```
+
+</a>
```
Comment (srveit) [inline Backup/backup-anywhere.md]: Remove this line

**Comment by jim-phillips** [comment]
@kyle-miller-centurylink please address the comments

**Comment by jim-phillips** [comment]
closing due to stale activity

**Pull request closed without merging**